app/gui/view/graph-editor/src/lib.rs

```diff
@@ -681,6 +681,7 @@ ensogl::define_endpoints_2! {
         node_position_set         ((NodeId,Vector2)),
         node_position_set_batched ((NodeId,Vector2)),
         node_expression_set       ((NodeId,ImString)),
+        node_port_expression_set  ((NodeId,span_tree::Crumbs,ImString)),
```

Comment:
missing space after `,`. I know that in other places the same mistake is made (this is from old times, when we were not using auto-formatter).